versions/3.1.0.md

```diff
@@ -1533,6 +1533,8 @@ When passing complex objects in the `application/x-www-form-urlencoded` content
 
 It is common to use `multipart/form-data` as a `Content-Type` when transferring request bodies to operations.  In contrast to 2.0, a `schema` is REQUIRED to define the input parameters to the operation when using `multipart` content.  This supports complex structures as well as supporting mechanisms for multiple file uploads.
 
+In a `multipart/form-data` request body, each property, or each element of an array property, takes a section in the payload with an internal header. For example, for the header `Content-Disposition: form-data; name="submit-name"`, the name of the property is `submit-name`. This type of form property is more commonly used for file transfers. The serialization strategy for each property of a `multipart/form-data` request body can be specified in an associated [`Encoding Object`](#encodingObject).
```

Comment:
Apart from the last sentence, this new paragraph seems to be summarising the defined behaviour of the `multipart/form-data` spec, not defining how the OAS works in conjunction with it. Is it needed?

Reply:
I agree.  I don't think it is needed.

Reply:
I am open to modifying or even removing this paragraph, but let me make a few points:

- I mostly borrowed the wording for this paragraph from the OpenAPI 2.0 spec [here](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md#parameter-object). So maybe this was also not needed in the 2.0 spec, but I personally found it helpful and thought we should move it forward to 3.0.

- In particular, I think there is value in describing how schema property values map to form data / payload sections. You may consider this "obvious", but I think it is better for a specification to state the obvious than to leave things undefined.

- In multiple conversations I’ve had on this issue, including in the last TSC, I think there has been agreement that the current spec needs to be clarified on how multipart/form-data is handled.  This was my attempt to address that.

- By "the multipart/form-data spec", do you mean RFC 7578?  If so, then at the very least I would like to replace this with a reference to that RFC (currently not referenced anywhere in the spec).

So should I take a stab at revising this paragraph or are folks still opposed to providing any explanation/clarification on this?

Reply:
I missed the fact that you were explaining how the Schema property maps to a parameter in multipart/form-data.   I think that aspect is worth clarifying.  I think it is important to be explicit about the form-data parameter mapping to the corresponding *schema* property.